src/PIL/Image.py

```diff
@@ -38,6 +38,7 @@
 from collections.abc import Callable, MutableMapping
 from enum import IntEnum
 from pathlib import Path
+from typing import List, NamedTuple, Tuple
```

Comment:
Let's add `from __future__ import annotations` and then:
```suggestion
from typing import NamedTuple
```